src/main/steps/applicant1/hub-page/sole/hub.njk

```diff
@@ -12,7 +12,9 @@
   <p class="govuk-body">{{ d8Awaiting.line2 }}</p>
   <p class="govuk-body">{{ d8Awaiting.line3 }}</p>
 {% elif progressionIndex == 8 %}
-  {% if isSuccessfullyServedByBailiff %}
+  {% if isDeemedOrDispensedApplication %}
+    <p class="govuk-body">{{ holdingAndDeemedOrDispensedAccepted.line1 | safe }}</p>
```

Comment:
same as above, is it possible to move the anchor tag here rather than it being in the content file?